src/csuite/input-number/InputNumberUI.tsx

```diff
@@ -418,12 +419,15 @@ export const InputNumberUI = observer(function InputNumberUI_(p: InputNumberProp
                      {p.text && (
                         <div /* Inner Label Text - Not shown while editing */
                            tw={['z-10 w-full truncate border-0 border-transparent pr-1 text-left outline-0']}
+                           style={{ fontSize: `${theme.inputText}pt` }}
```

Comment:
we should just make it so inputs inherit by default;
I think we shouldn't have more than once place where we need to define the font-size. 

Also, I like that we have variable font-size detached from interface size.

Reply:
We're going to need to do this to split group/widget header/ widget text options anyways.